idunn/utils/prometheus.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+from apistar import Component
+from prometheus_client import Counter, Gauge, Histogram
+
+
+WIKI_REQUEST_DURATION = Histogram(
+    "wiki_request_duration_seconds",
+    "Time spent processing a Wiki request.",
+    ["handler"],
+)
+
+WIKI_BREAKER_STATUS = Gauge(
+    "breaker_status",
+    "Breaker status, open (=2) or half-open (=1) or close (=0).",
+    []
+)
+
+WIKI_BREAKER_EXCEPTIONS_COUNT = Counter(
+    "breaker_exceptions_count",
```

Comment:
I think all prometheus metrics should be prefixed by `idunn_`

Reply:
done